<launch> 
	<node pkg="tf" type="static_transform_publisher" name="map_utm_odom_tf" args= "0.0 0.0 0.0  0.0 0.0 0.0 map odom 10" /> 
   
	<node name="utm_odom_node" pkg="utm_to_odom" type="utm_odom_node" >  <!--  output="screen" -->
		<param name="use_two_gps"             type = "bool"              value = "true"/>        
		<param name="use_gps_init_datum"      type = "bool"              value = "false" />    
		<param name="waypoint_file_name"      type = "string"            value = "/home/kyeongseo/one_fifth_catkin_ws/waypoints/r.txt"/>
		
   </node>     

</launch> 
